modules/cnv-understanding-virtio-drivers.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc
+
+[id="cnv-understanding-virtio-drivers_{context}"]
+= Understanding VirtIO drivers
+
+VirtIO drivers are paravirtualized device drivers required for Microsoft Windows
+ virtual machines to run properly in {ProductName}. The supported drivers are 
+available in the `virtio-win` container disk of the Red Hat Container Catalog.
```

Comment:
@aburdenthehand I agree with @kalexand-rh the link is https://access.redhat.com/containers/#/product/5be1983a5a13463a3e1d8ef4. It will make reading the guide much easier.